docs/versioned/admin/admin-overview.md

```diff
@@ -50,59 +96,61 @@ Knative configurations are performed by the following methods:
 
 - Using the Knative Operator
 
-    Some platform-wide settings can be managed declaratively using the Knative Operator.
-
-### Configuration tasks
-
-Knative documentation provides the following configuration procedures. This list is subject subject to change.
+    Some platform-wide settings can be managed declaratively using the Knative Operator, installed with the `kn` Knative CLI plugin. You can manage the operator without using the `kn` CLI. The `kn` CLI manages only operator installations.
 
-Configurations for default settings:
 
-- [Broker defaults](../eventing/configuration/broker-configuration.md)
-- [ConfigMap defaults](../serving/configuration/config-defaults.md)
-- [Event source defaults](../eventing/configuration/sources-configuration.md)
-- [Channel defaults](../eventing/configuration/channel-configuration.md)
-- [Kafka channel defaults](../eventing/configuration/kafka-channel-configuration.md)
-- [Domain names](../serving/using-a-custom-domain.md)
-- [Ingress gateway replacement](../serving/setting-up-custom-ingress-gateway.md)
+For more information, see [Installing CLI tools](../client/README.md)
 
-Configurations for new development:
-
-- [Deployment resources](../serving/configuration/deployment.md)
-- [Istio access to deployed services](../serving/istio-authorization.md)
-- [Namespace exclusion from webhooks](../serving/istio-authorization.md)
+Knative uses Kubernetes YAML manifests to define and configure system components. These manifests include core resources, custom resource definitions (CRDs), and extensibility features. As with Kubernetes, these configuration resources are declarative and can be managed using the `kubectl` CLI tool or with continuous delivery tools.
 
-Configurations for maintenance:
+## Configuration summaries
 
-- [Garbage collection](../serving/revisions/revision-admin-config-options.md)
-- [High availability](../serving/config-ha.md)
-- [Rollout duration for revisions](../serving/configuration/rolling-out-latest-revision-configmap.md)
-- [Autoscaling of Kafka features](../eventing/configuration/keda-configuration.md)
+The following sections provide an overview of the current configurations procedures of interest to Administrators. You make these configurations using `kubectl`, with some procedures applying YAML files.
 
-Configurations for security encryptions:
+### Networking
```

Comment:
Since networking is a sub-facet of serving, I'd probably put "Serving" first in the list (or you can alphabetize the sections, putting "Eventing" first).